bindings/cpp/README.md

```diff
@@ -54,15 +60,45 @@ This should output
 
 ## Building the C++ wrapper
 
+Dependencies: `python3-dev python3-pybind11`  (ubuntu package name)
+
+### Unix Makefile
 Now build the C++ test program:
 
 ```
 cd bindings/cpp
 make
 ```
+## CMake
+
+A CMakeLists.txt has been provided, it can detect system installation of pybind11, or download the latest pybind11 as a submodule. 
```

Comment:
I thought I had solved all these problems in the Makefile. The Makefile uses the current "python3" executable on the PATH to find the path to pybind11. It is tested on Ubuntu 18.04 via the CI workflow. The libraries you need to link with are also detected in the Makefile using python3-config.  The only issue I'm aware of is the problem with Python 3.8 where you *need* to specify --embed to python3-config, and in older pythons, you *can't* specify --embed (i.e. it is totally backwards-incompatible, as usual for Python).  When this came up, my plan would have been to detect explicitly Python >= 3.8 and add the --embed flag.

Reply:
this info  deserves to make into Readme, Unix makefile section?